controller/common/customize/hook.go

```diff
@@ -0,0 +1,40 @@
+package customize
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	v1alpha1 "metacontroller.io/apis/metacontroller/v1alpha1"
+	"metacontroller.io/hooks"
+)
+
+type CustomizableController interface {
+	GetCustomizeHook() *v1alpha1.Hook
+}
+
+// RelatedHookRequest is the object sent as JSON to the related hook.
```

Comment:
nit: I think this dates back to when this hook was named differently

Reply:
thanks, removed comments for now